src/preferences/constants.h

```diff
@@ -17,4 +17,12 @@ enum class ScreenSaverPreference {
     PREVENT_ON_PLAY = 2
 };
 
+enum class MultiSamplingMode {
```

Comment:
so to fix the conversion warnings, you'll need to add a `Q_ENUM_NS(MultiSamplingMode)` after the definition of this enum and in order to make that work, you'll need to make the file be processed by the Qt moc, by adding a `Q_NAMESPACE` below the `namespace mixxx` line.